README.md

```diff
@@ -89,6 +89,7 @@ export default MyNode;
     - [Full Reference](#full-reference)
     - [Alternative Hotkeys](#alternative-hotkeys)
     - [Specifying key events (keydown, keypress, keyup)](#specifying-key-events-keydown-keypress-keyup)
+    - [Specifying key map display data](#specifying-key-map-display-data)
```

Comment:
🐥 Updates readme to mention new syntax for defining key maps and new format returned by `getApplicationKeyMap()`